### PR TITLE
Add the new branch with the trial files

### PR DESCRIPTION
**Pull request by Peter1907**
Adding the HTML & CSS for the project

### CODE REVIEW

**Comment by Peter1907** [comment]
Hi, @peter1907
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes

Check the comments under the review.
Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Peter1907** [comment]
Good use of semantics but with many empty html tags

**Comment by Peter1907** [comment]
Kindly ensure that you have a descriptive README file & make sure that you set up linters correctly

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Peter1907 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good design
✔️  Good use of polite language

# Required Changes ♻️
- [ ] @Peter1907 Kindly ensure to add inline comments to your review, the comment should be in the section you have identified an error. Also, explain why the changes have to be made.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <li></li>
+        <li></li>
+        <li></li>
+      </ul>
+    </section>
+    <section id="Cell_Phones" class="main-section">
+      <h4>Cell Phones</h4>
+      <p></p>
+      <p></p>
+      <p></p>
```
Comment (Peter1907) [inline index.html:56]: Good use of semantics but make sure not to leave empty html tags

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#navbar {
+    position: fixed;
+    left: 0;
+}
+@media screen and (min-width: 480px) {
+    body {
+      background-color: lightgreen;
+    }
+  }
```
Comment (Peter1907) [inline css/style.css:9]: Kindly make sure you have an empty line at the end of your css file

**Review by algerina — APPROVED** [review]
Hi  @Peter1907 , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- You provided a good review using inline comments:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**